experiment/simulation/simulation.html

```diff
@@ -0,0 +1,319 @@
+							</svg>
+							<div class="square green">1</div>
+						</div>
+					</div>
+					<div id="one-zero">
+						<div id="path1">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,0 78.032901,0.75866 c 0,0 46.153515,-1.85739 43.310645,45.26492 v 7,0 l 52.37123,-0.48233" />
+							</svg>
+							<div class="square blue">0</div>
+						</div>
+						<div class="square black xor">XOR
+						</div>
+						<div id="path2">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,0 48,0 48,110 78.032901,110.75866 c 0,0 46.153515,-1.85739 43.310645,45.26492 v 7,0 l 52.37123,-0.48233" />
+							</svg>
+							<div class="square orange">0</div>
+						</div>
+						<div id="path3">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,100 38,100 38,210 78.032901,210.75866 c 0,0 46.153515,1.85739 43.310645,-45.26492" />
+							</svg>
+							<div class="square red">1</div>
+						</div>
+						<div class="square black and">AND
+						</div>
+						<div id="path4">
+							<svg width="256" height="256" viewBox="0 0 256 256">
+								<path fill="none" stroke="currentColor" stroke-width="1"
+									d="M 13.544914,100 78.032901,100 c 0,0 46.153515,1.85739 43.310645,-45.26492" />
+							</svg>
+							<div class="square green">1</div>
+						</div>
+						<!-- <div id="pp">
+				<button class="play">Play</button>
+				<button class="pause">Pause</button>
+			  </div> -->
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="column is-3-desktop is-4-tablet is-12-mobile">
+			<div class="v-datalist-container components-list">
+				<div class="v-datalist-title">Observations</div>
+				<div class="v-datalist-row">
+					<div class="v-table-wrap">
+						<table class="table is-bordered is-fullwidth">
+							<thead id="table-head">
+								<tr>
+									<th colspan="2">Inputs</th>
+									<th colspan="2">Expected Values</th>
+								</tr>
+								<tr>
+									<th>A</th>
+									<th>B</th>
+									<th>Sum</th>
+									<th>Carry</th>
+								</tr>
+							</thead>
+							<tbody id="table-body">
+								<tr>
+									<th>0</th>
+									<th>0</th>
+									<th>0</th>
+									<th>0</th>
+								</tr>
+								<tr>
+									<th>1</th>
+									<th>0</th>
+									<th>1</th>
+									<th>0</th>
+								</tr>
+								<tr>
+									<th>1</th>
+									<th>1</th>
+									<th>0</th>
+									<th>1</th>
+								</tr>
+								<tr>
+									<th>0</th>
+									<th>1</th>
+									<th>1</th>
+									<th>0</th>
+								</tr>
+							</tbody>
+						</table>
+					</div>
+				</div>
+
+			</div>
+		</div>
+	</div>
+
+</body>
+<script src="js/jsplumb.bundle.js"></script>
```

Comment:
Please keep script tags inside the body element.
Please load libraries from a cdn with exact versions and preferably with verification of their hash code. Do not store them locally.

Reply:
Yes, moved it inside the body element. I am still facing difficulties with loading jsPlumb as a cdn, so it has been stored locally.